packages/ckeditor5-source-editing/README.md

```diff
@@ -1,4 +1,20 @@
 CKEditor 5 source editing feature
-=================================
+========================================
 
-This is an initial package for development purposes. It does not contain code yet.
+[![npm version](https://badge.fury.io/js/%40ckeditor%2Fckeditor5-source-editing.svg)](https://www.npmjs.com/package/@ckeditor/ckeditor5-source-editing)
+[![Dependency Status](https://david-dm.org/ckeditor/ckeditor5-source-editing/status.svg)](https://david-dm.org/ckeditor/ckeditor5-source-editing)
+[![devDependency Status](https://david-dm.org/ckeditor/ckeditor5-source-editing/dev-status.svg)](https://david-dm.org/ckeditor/ckeditor5-source-editing?type=dev)
```

Comment:
Recently we removed these badges. Please, replace those with the following.

```suggestion
[![Coverage Status](https://coveralls.io/repos/github/ckeditor/ckeditor5/badge.svg?branch=master)](https://coveralls.io/github/ckeditor/ckeditor5?branch=master)
[![Build Status](https://travis-ci.com/ckeditor/ckeditor5.svg?branch=master)](https://travis-ci.com/ckeditor/ckeditor5)
```

Reply:
Done.